conductr_cli/test/test_conduct_load_v2.py

```diff
@@ -0,0 +1,114 @@
+            'configuration': None
+        }
+
+        self.default_url = 'http://127.0.0.1:9005/v2/bundles'
+
+        self.default_files = [
+            ('bundleConf', ('bundle.conf', 'mock bundle.conf')),
+            ('bundle', ('bundle.zip', 1))
+        ]
+
+    def test_success(self):
+        zip_entry_mock = MagicMock(return_value='mock bundle.conf')
+        with patch('conductr_cli.bundle_utils.zip_entry', zip_entry_mock):
+            self.base_test_success()
+        zip_entry_mock.assert_called_with('bundle.conf', self.bundle_file)
+
+    def test_success_verbose(self):
+        zip_entry_mock = MagicMock(return_value='mock bundle.conf')
+        with patch('conductr_cli.bundle_utils.zip_entry', zip_entry_mock):
+            self.base_test_success_verbose()
+        zip_entry_mock.assert_called_with('bundle.conf', self.bundle_file)
+
+    def test_success_long_ids(self):
+        zip_entry_mock = MagicMock(return_value='mock bundle.conf')
+        with patch('conductr_cli.bundle_utils.zip_entry', zip_entry_mock):
+            self.base_test_success_long_ids()
+        zip_entry_mock.assert_called_with('bundle.conf', self.bundle_file)
+
+    def test_success_custom_ip_port(self):
+        zip_entry_mock = MagicMock(return_value='mock bundle.conf')
+        with patch('conductr_cli.bundle_utils.zip_entry', zip_entry_mock):
+            self.base_test_success_custom_ip_port()
+        zip_entry_mock.assert_called_with('bundle.conf', self.bundle_file)
+
+    def test_failure(self):
+        zip_entry_mock = MagicMock(return_value='mock bundle.conf')
+        with patch('conductr_cli.bundle_utils.zip_entry', zip_entry_mock):
+            self.base_test_failure()
+        zip_entry_mock.assert_called_with('bundle.conf', self.bundle_file)
+
+    def test_failure_invalid_address(self):
+        zip_entry_mock = MagicMock(return_value='mock bundle.conf')
+        with patch('conductr_cli.bundle_utils.zip_entry', zip_entry_mock):
+            self.base_test_failure_invalid_address()
+        zip_entry_mock.assert_called_with('bundle.conf', self.bundle_file)
+
+    def test_failure_no_bundle(self):
+        self.base_test_failure_no_bundle()
+
+    def test_failure_no_bundle_conf(self):
+        urlretrieve_mock = MagicMock(return_value=(self.bundle_file, ()))
+        zip_entry_mock = MagicMock(return_value=None)
+        stderr = MagicMock()
+
+        with patch('conductr_cli.conduct_load.urlretrieve', urlretrieve_mock), \
+            patch('sys.stderr', stderr), \
+            patch('conductr_cli.bundle_utils.zip_entry', zip_entry_mock):
+            args = self.default_args.copy()
+            conduct_load.load(MagicMock(**args))
+
+        self.assertEqual(
+            as_error(strip_margin("""|Error: Problem with the bundle: Unable to find bundle.conf within the bundle file
+                                     |""")),
+            self.output(stderr))
+        zip_entry_mock.assert_called_with('bundle.conf', self.bundle_file)
```

Comment:
None of the base test methods being called here. Is that expected (why have the base here then)?